package.json

```diff
@@ -61,6 +61,7 @@
     "react-redux": "^7.1.1",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.1.2",
+    "react-swipe-component": "^3.0.0",
```

Comment:
Dlaczego użyłeś akurat ten komponenet? Od 2 lat nie jest rozwijany i może być problematyczny w integracji, co widać w przypadku testów

Reply:
na stronie npm tego dodatku, statystyki pokazują że jest on cały czas popularny. Dlatego go załadowałem, pierwszy jaki znalazłem. Po przeczytaniu Twoich uwag jednak zrobiłem głębsze rozeznanie i znalazłem coś nowszego - swipera. Zrobiłem z nim przechodzenie po stronach. Powinno być lepiej.

Reply:
Maciek, nie lepiej napisać swojego swipa? to nie jest duży komponent, jak co mogę w wolnej chwili pomóc. Gotowce często trudno dostosować do projektu, który nie wiadomo jak się dalej rozwinie.